docs/sources/upgrade-guide/when-to-upgrade/index.md

```diff
@@ -0,0 +1,115 @@
+
+## What to expect from each release type
+
+We split Grafana OSS and Grafana Enterprise releases into three main categories:
+
+- **Minor release (every other month)**: These releases can include new features, deprecation notices, notices about upcoming breaking changes, previously announced breaking changes, bug fixes, and security vulnerability patches
+- **Major release (once a year, in April/May)**: Like a minor release but accompanied by [GrafanaCON](https://grafana.com/events/grafanacon/) and a comprehensive upgrade guide for users who like to upgrade only once a year
+- **Patching release (every month)**: These include bug fixes for currently supported versions, as well as any security vulnerability patches
+
+You can choose your cadence: For frequent self-managed updates, you should follow the minor release (for example, upgrade 10.1 to 10.2), which also gives you access to the latest features. If you need a longer period to review our new releases, you should follow the major releases. Both strategies get patching releases with security fixes (high severity security fixes also result in ad-hoc patch releases). We’ll get into additional guidance on upgrade cadences later in this guide.
+
+## How to find the specifics for a release
+
+We love sharing all our great features with you so you can leverage Grafana to its fullest. We also understand that great release documentation allows you to upgrade with confidence.
+Whether it’s knowing that a pesky bug has been fixed, seeing that a security vulnerability is patched, or understanding how to mitigate the impact of breaking changes, proper documentation allows you to make informed decisions about when to upgrade your local Grafana instances.
+
+We provide release documentation in multiple places to address different needs:
+
+- [**What’s new**](https://grafana.com/docs/grafana/latest/whatsnew/?pg=blog&plcmt=body-txt) outlines new features debuting in each major and minor release.
+- [**Breaking changes**](https://grafana.com/docs/grafana/latest/breaking-changes/?pg=blog&plcmt=body-txt) notify you of updates included in major releases that could impact you and provide mitigation recommendations when needed.
+- [**Upgrade guides**](https://grafana.com/docs/grafana/latest/upgrade-guide/?pg=blog&plcmt=body-txt) instruct you on how to upgrade to a newer minor or major version.
+- And finally, a [**changelog**](https://github.com/grafana/grafana/blob/main/CHANGELOG.md) is generated for every release (major, minor, patching, security) and outlines all changes included in that release.
+
+## When to expect releases
+
+Currently, Grafana is on a monthly release cycle, with release types rotating every other month between minor or patching. For example, here’s a look at what scheduled releases to expect for the remainder of 2024:
```

Comment:
This description doesn't align with what's in the table or the **What to expect from each release type** section. I'm not seeing a rotation of any kind. We do patching monthly, minor every other month:

> with release types rotating every other month between minor or patching.